web/src/app/models/game.ts

```diff
@@ -93,4 +93,48 @@ export class Game {
       const answer = new Entry({ tiles, clue });
     });
   }
+
+  private printRow(row: Tile[]): string {
+    let rowStr = '';
+    row.forEach((tile: Tile) => {
+      if (tile.value === -1) {
+        rowStr += '#';
+      } else if (tile.value === '') {
+        rowStr += ' ';
+      } else {
+        if (tile.correct) {
+          rowStr += tile.value;
+        } else {
+          rowStr += ' ';
+        }
+      }
+    });
+    return rowStr;
+  }
+
+  public serializeBoard(): string {
+    let boardStr = '';
+    this.board.forEach((row: Tile[]) => {
+      boardStr += this.printRow(row);
+    });
+    return boardStr;
```

Comment:
Use .map.

`this.board.map(() => this.printRow(row)).join('')`

Reply:
agreed